### PR TITLE
[WebProfilerBundle] deprecated import/export commands

### PR DESCRIPTION
**Pull request by fabpot**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | yes |
| Tests pass? | yes |
| Fixed tickets | part of #11742 |
| License | MIT |
| Doc PR | n/a |

### CODE REVIEW

**Comment by Tobion** [comment]
Shouldn't the import() and export() methods of the profiler be deprecated as well? https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/HttpKernel/Profiler/Profiler.php#L141

**Comment by Tobion** [comment]
I agree the import/export can be removed. I never used it myself. And if somebody needs similar functionality, he can implement it himself easily as he has access to all methods needed.

**Comment by fabpot** [comment]
We agreed on deprecated the commands, not the methods, but I think it makes sense. I will do that now.

**Comment by fabpot** [comment]
done

File: src/Symfony/Component/HttpKernel/Profiler/Profiler.php [hunk 1]
```diff
@@ -149,6 +151,8 @@ public function export(Profile $profile)
      * @param string $data A data string as exported by the export() method
      *
      * @return Profile A Profile instance
+     *
+     * @deprecated since Symfony 2.8, to be removed in 3.0.
```
Comment (stof) [inline src/Symfony/Component/HttpKernel/Profiler/Profiler.php]: deprecation warnings should be triggered

File: src/Symfony/Component/HttpKernel/Profiler/Profiler.php [hunk 2]
```diff
@@ -137,6 +137,8 @@ public function purge()
      * @param Profile $profile A Profile instance
      *
      * @return string The exported data
+     *
+     * @deprecated since Symfony 2.8, to be removed in 3.0.
```
Comment (Tobion) [inline src/Symfony/Component/HttpKernel/Profiler/Profiler.php]: Please add the trigger deprecation code.

File: src/Symfony/Component/HttpKernel/Profiler/Profiler.php (hunk 1 above)
Comment (fabpot) [inline src/Symfony/Component/HttpKernel/Profiler/Profiler.php]: done

File: src/Symfony/Component/HttpKernel/Profiler/Profiler.php (hunk 2 above)
Comment (fabpot) [inline src/Symfony/Component/HttpKernel/Profiler/Profiler.php]: done

**Comment by Tobion** [comment]
Tests are failing.

**Comment by sstok** [comment]
:+1: for deprecating/removing this.

**Comment by fabpot** [comment]
tests fixed now

**Pull request merged**